packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -48,7 +48,7 @@ class BuildIOSFrameworkCommand extends BuildSubCommand {
     usesTargetOption();
     usesFlavorOption();
     usesPubOption();
-    usesDartDefines();
+    usesDartDefinesOption();
```

Comment:
nit: the option is "dart-define" not "dart-defines", so maybe `usesDartDefineOption`?

Reply:
Fixed